test/e2e/instance-networking.e2e.ts

```diff
@@ -122,4 +122,9 @@ test('Instance networking tab — External IPs', async ({ page }) => {
 
   // And that button shouldbe enabled again
   await expect(page.getByRole('button', { name: 'Attach floating IP' })).toBeEnabled()
+
+  // Detach the ephemeral IP
+  await clickRowAction(page, 'ephemeral', 'Detach')
+  await page.getByRole('button', { name: 'Confirm' }).click()
+  await expect(externalIpTable.getByRole('cell', { name: 'ephemeral' })).toBeHidden()
```

Comment:
add an assert before clicking the row action that this same selector is visible. The `clickRowAction` implicitly tests for that, but I like to use the same selector for before-and-after checks.